packages/engine-server/lib/helpers.js

```diff
@@ -24,17 +24,12 @@ const bookingToNotification = (booking) => ({
   booking,
 })
 
-const transportToNotification = (transport) => ({
-  /* We currently
-   * only have notifications for when a transport
-   * has been created, hence the 'success' severity status
-   * and the 'new' event.status.
-   */
-  severity: severityByEventStatus(transport.status),
+const transportToNotification = (transport, event) => ({
```

Comment:
This would probably be nicer if the arguments was declared better, from a caller's perspective, it's hard to know that event is going to be used first then fallback to transport.status

but that can be refactored later (I GUESS)

Reply:
I.e the caller should probably always specify what event it is concerning and that should be what's being used. I also don't know why we send the whole transport object to the front end, when it's just going to show the event text and ID of the vehicle?